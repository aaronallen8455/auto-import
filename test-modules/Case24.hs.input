module Case24 where

main :: IO ()
main = P
