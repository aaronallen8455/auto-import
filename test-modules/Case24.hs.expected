module Case24 where

import Namespace (pattern P)

main :: IO ()
main = P
